pinot-tools/pom.xml

```diff
@@ -123,6 +123,10 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
I commented 2 times on this change. Those comments got marked as resolved without any change! We `should not` touch anything on `pinot-tools/pom.xml`. Please revert the change back.

Reply:
So weird, I specifically remember fixing this exact thing. I'll go back and fix it again, hopefully it will actually work this time.

Reply:
I think the same thing is happening here as above. I'm looking at the file and there does not seem to be anything wrong with the indentation, and I'm not sure how this diff was created in the first place.